.github/workflows/check-backport-changelog.yml

```diff
@@ -31,6 +31,7 @@ jobs:
             - name: 'Fetch relevant history from origin'
               run: git fetch origin ${{ github.event.pull_request.base.ref }}
             - name: Check CHANGELOG status
+              if: ${{ !contains(github.event.pull_request.labels.*.name, 'No Core sync required') && !contains(github.event.pull_request.labels.*.name, 'Backport from WordPress Core') }}
```

Comment:
Are there other cases where no core sync is required, or is it just backports from core?

Reply:
And a nit: could we title case the label? All other labels are, and most likely someone will, which will break the test

Reply:
Alternatively we could remove the label if you think there's no other types of changes except for backports from core. I think in other cases we should probably add it to the excluded paths.

Reply:
> Are there other cases where no core sync is required, or is it just backports from core?

There are possibly several situations aside from backports:

1. Experimental code wrapped in `defined( 'IS_GUTENBERG_PLUGIN' )`
2. Experimental code in general :)
3. Adding `gutenberg` text domains to i18n translations (needed in the plugin, but not in Core)
4. Gutenberg hotfixes?
5. I'm out of ideas.... 😄

Reply:
> And a nit: could we title case the label? All other labels are, and most likely someone will, which will break the test

Good call. Will do, thanks!